src/solver/constraints-builder/cbuilder.h

```diff
@@ -140,8 +140,8 @@ class Cycle
         uint columnImpedance = (uint)Antares::Data::fhlImpedances;
 
         std::vector<double> impedances;
-        Data::AreaLink* previousLine;
-        double currentLineSign;
+        Data::AreaLink* previousLine = linkList[0]->ptr;
+        double currentLineSign = 1;
```

Comment:
It seems that those 2 variables weren't initialized before. I'm a bit surprised it didn't cause a segfault at line 161 for example.